.github/ISSUE_TEMPLATE/new_rule.yaml

```diff
@@ -0,0 +1,88 @@
+name: New Rule
+description: Suggestions and ideas for new rules
+title: "[New Rule] Name of rule"
+labels: ["Rule: New", "Team: TRADE"]
+assignees: []
+projects: ["elastic/1268"]
+
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: "Provide a detailed description of the activity to be detected."
+      placeholder: "Detailed description..."
+
+  - type: dropdown
+    id: target_ruleset
+    attributes:
+      label: Target Ruleset
+      description: "Select the target rulset."
+      options:
+        - apm
+        - cross-platform
+        - integrations
+        - linux
+        - macos
+        - ml
+        - network
+        - promotions
+        - threat_intel
+        - windows
+        - other
+
+  - type: dropdown
+    id: rule_type
+    attributes:
+      label: Target Rule Type
+      description: "Select the target type."
+      options:
+        - KQL
+        - ML_Job
+        - Threshold
+        - EQL
+        - Indicator_Match
+        - New_Terms
+        - ESQL
+
+  - type: input
+    id: ecs_version
+    attributes:
+      label: Tested ECS Version
+      description: "Specify the tested ECS version."
+      placeholder: "x.x.x"
+
+  - type: textarea
+    id: query
+    attributes:
+      label: Query
+      description: "Provide the query for the rule (optional)."
+      placeholder: "Query..."
+
+  - type: textarea
+    id: new_fields
+    attributes:
+      label: New fields required in ECS/data sources for this rule?
+      description: "List any new fields required in ECS or data sources for this rule (optional)."
+      placeholder: "New fields..."
+
+  - type: textarea
+    id: related_issues_prs
+    attributes:
+      label: Related issues or PRs
+      description: "Link any related issues or PRs (optional)."
+      placeholder: "Related issues or PRs..."
+
+  - type: textarea
+    id: references
+    attributes:
+      label: References
+      description: "List any references (optional)."
+      placeholder: "References..."
+
+  - type: textarea
+    id: example_data
```

Comment:
I'm on the fence about requiring this for a couple reasons.

- This is a public repository
- Data from certain integrations contains a vast amount of sensitive and non-sensitive information
- This requires the data to be redacted manually unless we have a tool for the user to use

Maybe there are better alternatives in the long term. Thoughts?

Reply:
It's optional. Maybe I should update this to "Redacted sample data" or something?

Reply:
As long as it's optional, should be good.